browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -1964,27 +1967,45 @@ L.CanvasTileLayer = L.Layer.extend({
 		this._map.fire('cellformula', {formula: formula});
 	},
 
+	_onCalcFunctionUsageMsg: function (textMsg) {
```

Comment:
in the followup PR we should move all these `_onCalcXXXXXX` functions into `_onMessage` in CalcTileLayer, as they are used only in Calc.